src/main/java/seedu/address/logic/commands/PreviousCommandHistoryCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Navigate to the previous command. Detected via up-arrow key.
+ */
+public class PreviousCommandHistoryCommand extends Command {
+    public static final String COMMAND_WORD = "upCommand";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the previous command entered. ";
+
```

Comment:
Code style: consider remove this line.

Reply:
Same goes here. Will update accordingly.